api/resolvers/item.js

```diff
@@ -1227,7 +1227,7 @@ export const createMentions = async (item, models) => {
 
         // only send if mention is new to avoid duplicates
         if (mention.createdAt.getTime() === mention.updatedAt.getTime()) {
-          notifyMention(user.id, item)
+          notifyMention({ models, userId: user.id, item })
```

Comment:
I considered altering the surrounding code in this context to only invoke this `notifyMention` function for users that didn't mute OP, but instead I kept that logic within `notifyMention` for consistency with the other `notifyXXX` functions. It's slightly dangerous to assume that the caller of `notifyXXX` is doing the mute filtering, IMO.

It would probably be more performant to do it here, instead of doing a DB call for each mentioned user, so I am open to refactoring if we think that's a major concern. But for consistency and all that, I did it this way.